include/common.sh

```diff
@@ -175,8 +175,8 @@ writeConfigFile(){
 # relative to this envvar value
 displayHelp() {
 echo "usage: . zopen-config [--eknv] [--knv] [-?|--help]"
-echo "  --override-zos-tools   Adds altbin/ dir to the PATH, which overrides /bin tools"
-echo "  --nooverride-zos-tools Does not add altbin/ dir from PATH"
+echo "  --override-zos-tools   Adds altbin/ dir to the PATH and altman/ dir to MANPATH, which overrides /bin tools"
```

Comment:
Not sure the trailing sentence _"..., which overrides..."_ makes sense now? Perhaps ", overriding the native z/OS tooling" (or similar words, not sure what the terminology is!)